src/components/typography/text/README.md

```diff
@@ -94,6 +94,7 @@ Wraps the given text in a `<p>` element, for normal content.
 | `children` | `PropTypes.node` |    ✅    | -                                                                             | -       |
 | `title`    | `String`         |    -     | -                                                                             | -       |
 | `truncate` | `Bool`           |    -     | -                                                                             | `false` |
+| `inline`   | `Bool`           |    -     | -                                                                             | `false` |
```

Comment:
`inline` or `isInline`? I am confused by the prop naming inconsistencies. When did it change and way if so from one convention to the other?

Reply:
Sorry, I copied the wrong thing and didn't even notice 🤦‍♂️

Reply:
I don't see this prop actually being used though? Shouldn't the default be "true" then?